### ISSUE TITLE
Razer Lancehead TE LEDs not working

### ISSUE THREAD

**Issue by t0msk** [openrazer/openrazer#1111]
### Distribution
Arch Linux - 5.6.2-arch1-2

openrazer-2.7.0-1

### Support

- [x] Have you tried rebooting? Yes
- [x] Have you looked at the [Troubleshooting](https://github.com/openrazer/openrazer/wiki/Troubleshooting) page? Yes

Log from `journalctl`:
```
Apr 23 10:32:15 tomsk-PC razer_mount[592]: Driver razermouse
Apr 23 10:32:15 tomsk-PC razer_mount[593]: Driver razermouse
Apr 23 10:32:15 tomsk-PC razer_mount[594]: Device_ID 0003:1532:0060.0005
Apr 23 10:32:15 tomsk-PC razer_mount[595]: Device_ID 0003:1532:0060.0007
Apr 23 10:32:15 tomsk-PC razer_mount[596]: Changing group /sys/bus/hid/drivers/razermouse/0003:1532:0060.0007/
Apr 23 10:32:15 tomsk-PC razer_mount[597]: Changing group /sys/bus/hid/drivers/razermouse/0003:1532:0060.0005/
Apr 23 10:32:15 tomsk-PC razer_mount[600]: Driver razermouse
Apr 23 10:32:15 tomsk-PC razer_mount[606]: Changed group /sys/bus/hid/drivers/razermouse/0003:1532:0060.0005/
Apr 23 10:32:15 tomsk-PC razer_mount[604]: Changed group /sys/bus/hid/drivers/razermouse/0003:1532:0060.0007/
Apr 23 10:32:15 tomsk-PC razer_mount[607]: Device_ID 0003:1532:0060.0006
Apr 23 10:32:15 tomsk-PC razer_mount[610]: Changing group /sys/bus/hid/drivers/razermouse/0003:1532:0060.0006/
Apr 23 10:32:15 tomsk-PC razer_mount[612]: Changed group /sys/bus/hid/drivers/razermouse/0003:1532:0060.0006/
```
Log from `~/.local/share/openrazer/logs/razer.log`:
```
2020-04-23 10:32:45 | razer                          | INFO     | Initialising Daemon (v2.7.0). Pid: 802
2020-04-23 10:32:45 | razer.screensaver              | INFO     | Initialising DBus Screensaver Monitor
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerAbyssus ---------------------------- (1532:0042)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerAbyssus1800 ------------------------ (1532:0020)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerAbyssus2000 ------------------------ (1532:005E)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerAbyssusV2 -------------------------- (1532:005B)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerAnansi ----------------------------- (1532:010F)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidow2019 --------------------- (1532:0241)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChroma ------------------- (1532:0203)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChromaOverwatch ---------- (1532:0211)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChromaTournamentEdition -- (1532:0209)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChromaV2 ----------------- (1532:0221)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowElite -------------------- (1532:0228)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowLite --------------------- (1532:0235)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowStealth ------------------ (1532:011B)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowStealthEdition ----------- (1532:010E)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowUltimate2012 ------------- (1532:010D)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowUltimate2013 ------------- (1532:011A)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowUltimate2016 ------------- (1532:0214)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowXChroma ------------------ (1532:0216)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowXTournamentEditionChroma - (1532:021A)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowXUltimate ---------------- (1532:0217)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlade2018 -------------------------- (1532:0233)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlade2018Base ---------------------- (1532:023B)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlade2018Mercury ------------------- (1532:0240)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlade2019Adv ----------------------- (1532:023A)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBlade2019Base ---------------------- (1532:0246)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeLate2016 ---------------------- (1532:0224)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeMid2019Mercury ---------------- (1532:0245)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladePro2017 ----------------------- (1532:0225)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladePro2017FullHD ----------------- (1532:022F)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeProLate2016 ------------------- (1532:0210)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeQHD --------------------------- (1532:020F)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealth ----------------------- (1532:0205)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealth2019 ------------------- (1532:0239)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealthLate2016 --------------- (1532:0220)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealthLate2017 --------------- (1532:0232)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealthLate2019 --------------- (1532:024A)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealthMid2017 ---------------- (1532:022D)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerChromaMugHolder -------------------- (1532:0F07)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerCore ------------------------------- (1532:0215)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerCynosaChroma ----------------------- (1532:022A)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdder1800 --------------------- (1532:0038)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdder2013 --------------------- (1532:0037)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdder3500 --------------------- (1532:0054)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdder3_5G --------------------- (1532:0016)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdderChroma ------------------- (1532:0043)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdderElite -------------------- (1532:005C)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathAdderEssential ---------------- (1532:006E)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathStalkerChroma ----------------- (1532:0204)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDeathStalkerExpert ----------------- (1532:0202)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerDiamondbackChroma ------------------ (1532:004C)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerFirefly ---------------------------- (1532:0C00)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerFirefly_Hyperflux ------------------ (1532:0068)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerGoliathus -------------------------- (1532:0C01)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerGoliathusExtended ------------------ (1532:0C02)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerHuntsman --------------------------- (1532:0227)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerHuntsmanElite ---------------------- (1532:0226)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerImperator -------------------------- (1532:002F)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerKraken71 --------------------------- (1532:0501)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerKraken71Alternate ------------------ (1532:0506)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerKraken71Chroma --------------------- (1532:0504)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerKraken71V2 ------------------------- (1532:0510)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerLanceheadTE ------------------------ (1532:0060)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerMamba2012Wired --------------------- (1532:0024)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerMamba2012Wireless ------------------ (1532:0025)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerMambaChromaWired ------------------- (1532:0044)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerMambaChromaWireless ---------------- (1532:0045)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerMambaTE ---------------------------- (1532:0046)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNaga2012 --------------------------- (1532:002E)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNaga2014 --------------------------- (1532:0040)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNagaChroma ------------------------- (1532:0053)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNagaHex ---------------------------- (1532:0041)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNagaHexRed ------------------------- (1532:0036)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNagaHexV2 -------------------------- (1532:0050)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNagaTrinity ------------------------ (1532:0067)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerNostromo --------------------------- (1532:0111)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrbweaver -------------------------- (1532:0113)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrbweaverChroma -------------------- (1532:0207)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrnata ----------------------------- (1532:021F)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrnataChroma ----------------------- (1532:021E)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrochi2011 ------------------------- (1532:0013)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrochi2013 ------------------------- (1532:0039)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOrochiWired ------------------------ (1532:0048)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerOuroboros -------------------------- (1532:0032)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerTaipan ----------------------------- (1532:0034)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerTartarus --------------------------- (1532:0201)
2020-04-23 10:32:45 | razer                          | DEBUG    | Loaded device specification: RazerTartarusChroma --------------------- (1532:0208)
2020-04-23 10:32:45 | razer                          | INFO     | Found device.0: 0003:1532:0060.0005
2020-04-23 10:32:45 | razer.device0                  | INFO     | Initialising device.0 RazerLanceheadTE
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.setDeviceMode method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getVidPid method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.suspendDevice method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.hasDedicatedMacroKeys method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getSerial method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDriverVersion method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.resumeDevice method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getRazerUrls method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDeviceMode method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getFirmware method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getMatrixDimensions method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.hasMatrix method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDeviceName method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDeviceType method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.dpi.maxDPI method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.dpi.getDPI method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.dpi.setDPI method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.getPollRate method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.misc.setPollRate method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.getLogoBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.getScrollBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.getLeftBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.getRightBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightBrightness method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoWave method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoStatic method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoSpectrum method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoNone method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoReactive method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoBreathRandom method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoBreathSingle method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.logo.setLogoBreathDual method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollWave method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollStatic method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollSpectrum method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollNone method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollReactive method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollBreathRandom method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollBreathSingle method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.scroll.setScrollBreathDual method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftWave method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftStatic method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftSpectrum method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftNone method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftReactive method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftBreathRandom method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftBreathSingle method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.left.setLeftBreathDual method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightWave method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightStatic method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightSpectrum method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightNone method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightReactive method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightBreathRandom method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightBreathSingle method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.right.setRightBreathDual method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setCustom method to DBus
2020-04-23 10:32:45 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setKeyRow method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.devices.getOffOnScreensaver method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.devices.supportedDevices method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.daemon.stop method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.daemon.version method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.devices.getDevices method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.devices.syncEffects method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.devices.enableTurnOffOnScreensaver method to DBus
2020-04-23 10:32:45 | razer                          | DEBUG    | Adding razer.devices.getSyncEffects method to DBus
2020-04-23 10:32:45 | razer                          | INFO     | Serving DBus
2020-04-23 10:32:47 | razer                          | DEBUG    | DBus called get_serial_list
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_device_type
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_device_name
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_device_type
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_firmware
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call version
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call has_matrix
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call has_matrix
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_dpi_xy
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call max_dpi
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_poll_rate
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_logo_brightness
2020-04-23 10:32:47 | razer.device0                  | DEBUG    | DBus call get_scroll_brightness
```

I don't know what happened, but suddenly my LEDs (all of them) are not working (they are off) anymore on my Razer Lancehead TE, but my mouse is working correctly only LEDs are off.
For frontend app I use `Polychromatic` and `RazerGenie` I tried to change effect and color of LEDs, but it didn't help, I tried both apps. My mouse is pretty new, it is only 4 months old.

**Comment by lah7** [comment]
It looks like the driver and daemon is working normally - have you tried changing the brightness?

I have a feeling that in `~/.config/openrazer/razer.conf`, you may have `devices_off_on_screensaver` set to `True`. It might be that the screensaver came on (or the system went to sleep) and upon unlocking, the brightness wasn't restored properly after unlock/resume. Try changing this to `False`.

Both Polychromatic 0.3.12 and RazerGenie 0.8.1 only support the logo/scroll LEDs at the moment. 

You can try setting them all to 100% brightness and a green static using the daemon directly by running this:
```
python3 -c "import openrazer.client; a = openrazer.client.DeviceManager().devices[0]; a.fx.misc.left.static(0,255,0); a.fx.misc.left.brightness = 100; a.fx.misc.right.static(0,255,0); a.fx.misc.right.brightness = 100; a.fx.misc.logo.static(0,255,0); a.fx.misc.logo.brightness = 100; a.fx.misc.scroll_wheel.static(0,255,0); a.fx.misc.scroll_wheel.brightness = 100;"
```
(Assumes this is the only OpenRazer device)

**Comment by t0msk** [comment]
Oh it is working :) Thank you :) Problem was that `devices_off_on_screensaver` was set to `True`, can I ask you one more question? Why I cannot change color of side LEDs in Polychromatic / RazerGenie? Looks like that in command line using your script I can :)

**Comment by lah7** [comment]
You're welcome! :+1:

The driver/daemon supports it but the frontends still have a bit of catch up to do.

I develop **Polychromatic** - I see you've already left an issue a few months ago: https://github.com/polychromatic/polychromatic/issues/225 :slightly_smiling_face:  I do have these zones implemented in a [`dev-1.0.0`](https://github.com/polychromatic/polychromatic/tree/dev-1.0.0) branch but it's quite buggy at the moment (see https://github.com/polychromatic/polychromatic/issues/244#issuecomment-618086850). This version depends on #888 being merged. Progress has been slow, but not abandoned! :snail: 

**RazerGenie** has a pull request open to add support: https://github.com/z3ntu/RazerGenie/pull/54

@nightsky30 has tweaked both stable version of the applications in a fork which you could use in the meantime:

* https://github.com/nightsky30/RazerGenie/tree/side-leds (in a pull request)
* https://github.com/nightsky30/polychromatic/tree/side-leds

**Comment by t0msk** [comment]
Ah I see :) RazerGenies pull request is about 1 year old, I wonder why it is not merged, when will be `dev-1.0.0` released? :)

**Comment by lah7** [comment]
I cannot say, development is unpredictable! :microscope:   "As soon as it's ready", some point this year? :crossed_fingers:
